rwgame/GameWindow.cpp

```diff
@@ -10,7 +10,7 @@ GameWindow::GameWindow() :
 
 void GameWindow::create(const std::string& title, size_t w, size_t h, bool fullscreen)
 {
-	uint32_t style = SDL_WINDOW_OPENGL | SDL_WINDOW_RESIZABLE;
+	uint32_t style = SDL_WINDOW_OPENGL | SDL_WINDOW_RESIZABLE | SDL_WINDOW_HIDDEN;
```

Comment:
Fair point. They're explicitly the same size but I'll change this to for consistency with what SDL2 uses.